our_site/planner/templates/planner/dashboard.html

```diff
@@ -37,6 +37,7 @@
     <div class="column">
         {% for event in event_list %}
         <div class="box">
+            <a href="{% url 'planner:event_home' event.id %}">{{ event.name }}</a>
```

Comment:
Since we already display the event name in the next line, could we merge these so the event's name only shows up once and isn't separately a link as well as a header?

Reply:
Or maybe we can keep it as a link and keep the event name 2x but make it look differently to the other event details so we can show they have different purposes? But I am good with either options